### DISCUSSION TITLE
[BUG] - error with Connect-PnPOnline

### DISCUSSION THREAD

**Discussion by DarylVoganMBI** [pnp/powershell#2555] [General]
Error connecting with a hard-coded non-MFA email and password

### Expected behavior 
successful conenction

### Actual behavior 
Connect-PnPOnline : There was an error parsing WS-Trust response from the endpoint. This may occur if there is an issue with your ADFS configuration. See https://aka.ms/msal-net-iwa-troubleshooting for 
more details. Error Message: Federated service at https://sts.xxxxxxxx.com/adfs/services/trust/2005/usernamemixed returned error: ID3242: The security token could not be authenticated or authorized. 

### Steps to reproduce behavior
cls
$siteUrl="https://xxxxx.sharepoint.com/sites/xxxxxx"
$spoUserName = "xxxxxxxxxxxx"
$spoUserPassword='xxxxxxxxxxxx'
$secPwd = $spoUserPassword | ConvertTo-SecureString -AsPlainText -Force
$creds = new-object -typename System.Management.Automation.PSCredential -argumentlist $spoUserName, $secPwd
$ReportFile="C:\Temp\DLA-ELD.csv"
remove-item $ReportFile -ErrorAction SilentlyContinue
$now=Get-Date
write-host "start - $now"
#$creds=Get-Credential
Connect-PnPOnline -URL $siteURL -Credentials $creds

### What is the version of the Cmdlet module you are running?
ModuleType Version    Name                                ExportedCommands                                                                                                                                  
---------- -------    ----                                ----------------                                                                                                                                  
Manifest   1.11.0     PnP.PowerShell                      {Add-PnPAdaptiveScopeProperty, Add-PnPPropertyBagValue, Add-PnPSiteClassification, Copy-PnPFolder...}                                             
Manifest   1.10.0     PnP.PowerShell                      {Add-PnPAdaptiveScopeProperty, Add-PnPPropertyBagValue, Copy-PnPFolder, Export-PnPClientSidePage...}  

### Which operating system/environment are you running PnP PowerShell on?
- [x ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by gautamdsheth** [comment]
Looks like some issue specific to your M365 environment , can you try with `-Interactive` parameter ?
Am moving this to discussion as it is not an issue with the module itself but some configuration which is interfering with the login process.

**Comment by JohnKoornneef** [comment]
After a failed password change, I got the same error. Resetting the password again solved this issue for me.
A "bad username or password" error would have been more helpful in this case.

**Comment by DarylVoganMBI** [comment]
Apologies, I should have followed up. Same thing, type snuck into the password. A better error message would have saved much effort.

From: JohnKoornneef ***@***.***>
Sent: Wednesday, November 16, 2022 3:37 AM
To: pnp/powershell ***@***.***>
Cc: Vogan, Daryl ***@***.***>; Author ***@***.***>
Subject: EXTERNAL: Re: [pnp/powershell] [BUG] - error with Connect-PnPOnline (Discussion #2555)


After a failed password change, I got the same error. Resetting the password again solved this issue for me.
A "bad username or password" error would have been more helpful in this case.

-
